lib/ruby/stdlib/ffi/platform/sysconf.rb.ffi

```diff
@@ -1,139 +0,0 @@
-module Platform;end
```

Comment:
jruby specific?

Reply:
I think these are template files that originally lived outside of the generation logic but now live in some form in the gen/ dir?